sdk/containerregistry/container-registry/package.json

```diff
@@ -84,7 +85,6 @@
     "@azure/core-rest-pipeline": "1.1.0-beta.4",
     "@azure/core-paging": "^1.1.1",
     "@azure/core-tracing": "1.0.0-preview.11",
-    "@azure/core-util": "^1.0.0-beta.1",
```

Comment:
I'm moving Key Vault to corev2 so figured it'll be useful info for me too - is there a reason not to use core-util? Like should I also make local copies of the methods I need from it? I think there's only like 2 functions there

Reply:
core-util is till in beta  and [we still want some investigation](https://github.com/Azure/azure-sdk-for-js/pull/15498#issuecomment-852459471). I am copying code for now so ACR GA isn't blocked on core-util beta version. We can move to core-util after it is in GA.